homeassistant/components/plex/strings.json

```diff
@@ -3,7 +3,7 @@
     "flow_title": "{name} ({host})",
     "step": {
       "user": {
-        "description": "Continue to [plex.tv](https://plex.tv) to link a Plex server."
+        "description": "Continue to [plex.tv]({api_url}) to link a Plex server."
```

Comment:
Since plex.tv is not translatable either maybe move the whole link out, see Tomorrow.io
https://github.com/home-assistant/core/pull/154759/files